test/core/util/tls_utils.h

```diff
@@ -43,6 +43,16 @@ PemKeyCertPairList MakeCertKeyPairs(absl::string_view private_key,
 
 std::string GetFileContents(const char* path);
 
+//  Checks that the string `private_key` is that of a key of supported type and
+//  in PKCS #8 syntax. Returns an OK status if so or a not-OK status otherwise.
+//  Logs supported but unexpected types.
+absl::Status CheckPrivateKey(absl::string_view private_key);
+
+//  Checks that the string `cert_chain` is that of a certificate chain
+//  containing certificates of supported type and PEM-encoded.
```

Comment:
X.509

Reply:
Changed it to
```
//  Checks that the string `cert_chain` is that of a certificate chain
//  containing PEM-encoded X.509 certificates.
//  Returns an OK status if so or a not-OK status otherwise.
```